hermes-frontend/build.gradle

```diff
@@ -7,7 +7,7 @@ dependencies {
     compile group: 'io.undertow', name: 'undertow-core', version: '1.4.3.Final'
     compile group: 'org.apache.kafka', name: 'kafka-clients', version: '0.8.2.0'
     compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.0'
-    compile group: 'net.openhft', name: 'chronicle-map', version: '2.4.13'
+    compile group: 'net.openhft', name: 'chronicle-map', version: '3.11.0'
```

Comment:
we need to make sure it's backwards compatible, so we're be able to read the backup storage file written with previous version.